test/server_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/cucumber/godog"
+
+	"github.com/SuperNerdIT/telepathic/cmd/server"
+)
+
+type serverCtxKey struct{}
+
+type statusCodeCtxKey struct {}
+
+func thisReturnsCode(ctx context.Context, expectedCode int) (error) {
```

Comment:
**Suggestion:**
This name could be something like `thisReturnsStatusCode`, so we can be more specific.
XD I had to read the whole file to understand what it was referring to.

Reply:
Sorry for that :P jaja , godog reads your .feature file and autogenerates a suggestion with the test file, and so we must change the feature and this function. I'd only want to tell you about that feature of godog